x/claim/keeper/msg_server_claim_eth.go

```diff
@@ -35,7 +35,7 @@ func (k msgServer) ClaimEth(goCtx context.Context, msg *types.MsgClaimEth) (*typ
 
 	// 3. validate signature
 	isValid, err := IsValidClaimSignature(msg.EthAddress, msg.Creator,
-		ethClaim.InitialClaimableAmount.AmountOf(types.DefaultClaimDenom).String(), msg.Signature)
+		ethClaim.InitialClaimableAmount.Amount.String(), msg.Signature)
```

Comment:
Based on the document , there will have claim record setup at genesis , I think those claim record will be useful for the first time user trying to claim. After first claim , we already create an additional claim record for `arkeo` chain,
So given that , instead of update the status as `ActionCompleted` , can we just delete it?  because we don't need it anymore .

The first 1/3 of token has been sent to their arkeo address, and the rest 2/3 will be credit to their arkeo address when they stake / vote.

Reply:
> So given that , instead of update the status as ActionCompleted , can we just delete it? because we don't need it anymore .

We could, i decided to leave the record so that if a user cam back to query it, they would see that they already claimed vs showing that there was nothing ever to claim.  Happy to delete if storage concerns on the node, etc make it worth removing the record